.github/workflows/nim_waku_daily.yml

```diff
@@ -0,0 +1,24 @@
+name: Nim Waku Interop Tests
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: false
+
+on:
+  schedule:
+    - cron:  '0 3 * * *'

```

Comment:
let's have them at different hours, EX: one at 0 3 and one at 0 4 
to avoid issues with shared resources while running workflows concurrently

Reply:
Changed in https://github.com/waku-org/waku-interop-tests/pull/13/commits/7f587399bb94952b1050c910acb5a7d9d9606333